### PR TITLE
docs: create nph.md

### PR DESCRIPTION
**Pull request by Ivansete-status**
## Description
Add instructions on how to properly configure the `nph` formatter.

### CODE REVIEW

File: docs/contributors/nph.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### Installation and configuration
+1. Ask the [nwaku team](https://discord.com/channels/1110799176264056863/1111541184490393691) about the required `nph` version.
+2. Download the desired release from _GitHub_ and place the binary in the PATH env var.
+3. Add the following content into `~/.config/Code/User/settings.json`:
+
+```
+{
+    "editor.renderWhitespace": "all",
+    "editor.largeFileOptimizations": false,
+    "[python]": {
```
Comment (alrevuelta) [inline docs/contributors/nph.md:20]: unrelated?

File: docs/contributors/nph.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "editor.formatOnType": true
+    },
+    "explorer.confirmDelete": false,
+    "explorer.confirmDragAndDrop": false,
+    "editor.rulers": [
+        100
+    ],
+    "diffEditor.ignoreTrimWhitespace": false,
+    "editor.parameterHints.enabled": false,
+    "rust-analyzer.inlayHints.parameterHints.enable": false,
```
Comment (alrevuelta) [inline docs/contributors/nph.md:30]: unrelated?

File: docs/contributors/nph.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "rust-analyzer.inlayHints.bindingModeHints.enable": true,
+    "rust-analyzer.inlayHints.closureCaptureHints.enable": true,
+    "editor.minimap.enabled": false,
+    "window.zoomLevel": 1,
+    "[nim]": {
```
Comment (alrevuelta) [inline docs/contributors/nph.md:35]: guess these are the only required lines?

**Review by NagyZoltanPeter — APPROVED** [review]
Great! Thanks for it.

**Review by SionoiS — APPROVED** [review]
LGTM
